lib/services/sf.js

```diff
@@ -615,6 +615,9 @@ function StateAbstract(options)
           // if we got an error, wrap it into a network error
           if (err)
           {
+            // if we're running in DEBUG loglevel, probably we want to see the full error instead
+            Logger.getInstance().debug(JSON.stringify(err, Util.getCircularReplacer()))
```

Comment:
could we add a prefixing message at the begining of this log? to not only log the reasult error as json

Reply:
ok